CMakeLists.txt

```diff
@@ -11,133 +11,135 @@
-  ELSE()
-    SET(CORSIX_TH_USE_FFMPEG OFF)
-    SET(CORSIX_TH_USE_LIBAV OFF)
-    MESSAGE("Note: FFMPEG video disabled since it requires SDL audio.")
-  ENDIF(WITH_AUDIO)
-ELSE()
-  SET(CORSIX_TH_USE_FFMPEG OFF)
-  SET(CORSIX_TH_USE_LIBAV OFF)
-  MESSAGE("Note: FFMPEG video is disabled")
-ENDIF(WITH_MOVIES)
-
-IF(WITH_FREETYPE2)
-  SET(CORSIX_TH_USE_FREETYPE2 ON)
-  MESSAGE("Note: FreeType2 is enabled (default)")
-ELSE()
-  SET(CORSIX_TH_USE_FREETYPE2 OFF)
-  MESSAGE("Note: FreeType2 is disabled")
-ENDIF(WITH_FREETYPE2)
-
-IF(MSVC)
-  IF(WITH_VLD)
-    SET(CORSIX_TH_USE_VLD ON)
-    MESSAGE("Note: Visual Leak Detector is enabled")
-  ELSE()
-    SET(CORSIX_TH_USE_VLD OFF)
-    MESSAGE("Note: Visual Leak Detector is disabled (default)")
-  ENDIF(WITH_VLD)
-ELSE()
-  SET(CORSIX_TH_USE_VLD OFF)
-ENDIF(MSVC)
+option(WITH_SDL "Activate SDL Renderer" ON) # our default option
+option(WITH_AUDIO "Activate Sound" ON) # enabled by default
+option(WITH_MOVIES "Activate in game movies" ON)
+option(WITH_FREETYPE2 "Enhanced Font Support" ON)
+option(WITH_LUAJIT "Use LuaJIT instead of Lua" OFF)
+option(WITH_LIBAV "Use LibAV instead of FFmpeg" OFF)
+option(BUILD_ANIMVIEWER "Build the animation viewer as part of the build process" OFF)
+if(MSVC)
+  option(WITH_VLD "Build with Visual Leak Detector for Visual Studio" OFF)
+endif()
+
+if(WITH_AUDIO)
+  set(CORSIX_TH_USE_SDL_MIXER ON)
+  message("Note: SDL audio is enabled (default)")
+else()
+  set(CORSIX_TH_USE_SDL_MIXER OFF)
+  message("Note: SDL audio is disabled")
+endif()
+
+if(WITH_MOVIES)
+  if(WITH_AUDIO)
+    if(WITH_LIBAV)
+      set(CORSIX_TH_USE_FFMPEG OFF)
+      set(CORSIX_TH_USE_LIBAV ON)
+      message("Note: LibAV video is enabled")
+    else()
+      set(CORSIX_TH_USE_FFMPEG ON)
+      set(CORSIX_TH_USE_LIBAV OFF)
+      message("Note: FFMPEG video is enabled (default)")
+    endif()
+  else()
+    set(CORSIX_TH_USE_FFMPEG OFF)
+    set(CORSIX_TH_USE_LIBAV OFF)
+    message("Note: FFMPEG video disabled since it requires SDL audio.")
+  endif()
+else()
+  set(CORSIX_TH_USE_FFMPEG OFF)
+  set(CORSIX_TH_USE_LIBAV OFF)
+  message("Note: FFMPEG video is disabled")
+endif()
+
+if(WITH_FREETYPE2)
+  set(CORSIX_TH_USE_FREETYPE2 ON)
+  message("Note: FreeType2 is enabled (default)")
+else()
+  set(CORSIX_TH_USE_FREETYPE2 OFF)
+  message("Note: FreeType2 is disabled")
+endif()
+
+if(MSVC)
+  if(WITH_VLD)
+    set(CORSIX_TH_USE_VLD ON)
+    message("Note: Visual Leak Detector is enabled")
+  else()
+    set(CORSIX_TH_USE_VLD OFF)
+    message("Note: Visual Leak Detector is disabled (default)")
+  endif()
+else()
+  set(CORSIX_TH_USE_VLD OFF)
+endif()
 
 # Get precompiled dependencies before running the various find modules
-if ( USE_PRECOMPILED_DEPS )
+if(USE_PRECOMPILED_DEPS)
   message("Note: Using precompiled dependencies.")
   include( PrecompiledDeps )
 endif()
 
 # Environment handling
-CHECK_INCLUDE_FILES(inttypes.h CORSIX_TH_HAS_INTTYPES_H)
+check_include_files(inttypes.h CORSIX_TH_HAS_INTTYPES_H)
```

Comment:
This can be dropped. C99 support was added in VS2013 so we don't need to handle this ourselves.